IFC4x3/Properties/i/InsulationResistance_1ZlGxOxPPFmwm2gfaKJrQQ/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="InsulationResistance_1ZlGxOxPPFmwm2gfaKJrQQ" Name="InsulationResistance" UniqueId="63bd0ed8-ed96-4fc3-ac02-aa99144f569a" PrimaryDataType="IfcElectricResistanceMeasure" />
```

Comment:
Could we generalize this and `Pset_SolidStratumCapacity.Resistivity`?

Reply:
Agree! Commit : b5e658cad21e4944423ed95d945268c722299c68